recipes/lz4/config.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Is there a specific reason why we start these files with `---`?

Reply:
```
YAML uses three dashes (“---”) to separate directives from document content. This also serves to signal the start of a document
if no directives are present. Three dots ( “...”) indicate the end of a document without starting a new one, for use in communication channels.
```

I think these are not needed but harmless.